src/libs/workflows-app-utils.js

```diff
@@ -0,0 +1,19 @@
+export const resolveRunningCromwellAppUrl = (apps, currentUser) => {
```

Comment:
i'd recommend making this small new file a `.ts` file, and leveraging the fact we have type safety on things like `App` and `AppStatus`! There's ca be some tricky nuances handling these complex objects that the types help a lot with, not least of which is changes related to the leonardo contract (aka returns of GET/LIST App). The code in its current state is very vulnerable to breaking if the underlying leo implementation breaks.

Reply:
I have converted both `workflows-app-utils` file and it its test file to TS